components/eventing-controller/reconciler/subscription-nats/reconciler.go

```diff
@@ -43,15 +43,15 @@ const (
 
 func NewReconciler(client client.Client, applicationLister *application.Lister, cache cache.Cache, log logr.Logger, recorder record.EventRecorder, cfg env.NatsConfig) *Reconciler {
 	natsClient := handlers.NewNats(cfg, log)
```

Comment:
One flavor, should be addressed in an extra PR: In Go acronyms are written all-caps. Also it's no client anymore, it's a handler backend. 
```suggestion
	backend := handlers.NewNATSBackend(cfg, log)
```

Reply:
Renamed "Client" to "handler" in both flavors (nats and beb)